x509/src/certificate.rs

```diff
@@ -33,160 +33,52 @@ impl Default for Version {
     }
 }
 
-/// X.509 `TBSCertificate` as defined in [RFC 5280 Section 4.1.2.5]
+/// X.509 `TBSCertificate` as defined in [RFC 5280 Section 4.1]
 ///
-/// ASN.1 structure containing the names of the subject and issuer, a public key associated
-/// with the subject, a validity period, and other associated information.
+/// ASN.1 structure containing the names of the subject and issuer, a public
+/// key associated with the subject, a validity period, and other associated
+/// information.
 ///
 /// ```text
-///   TBSCertificate  ::=  SEQUENCE  {
-///       version         [0]  Version DEFAULT v1,
-///       serialNumber         CertificateSerialNumber,
-///       signature            AlgorithmIdentifier{SIGNATURE-ALGORITHM, {SignatureAlgorithms}},
-///       issuer               Name,
-///       validity             Validity,
-///       subject              Name,
-///       subjectPublicKeyInfo SubjectPublicKeyInfo,
-///       ... ,
-///       [[2:               -- If present, version MUST be v2
-///       issuerUniqueID  [1]  IMPLICIT UniqueIdentifier OPTIONAL,
-///       subjectUniqueID [2]  IMPLICIT UniqueIdentifier OPTIONAL
-///       ]],
-///       [[3:               -- If present, version MUST be v3 --
-///       extensions      [3]  Extensions{{CertExtensions}} OPTIONAL
-///       ]], ... }
+/// TBSCertificate  ::=  SEQUENCE  {
```

Comment:
The brackets in comments should be escaped to make cargo doc happy.

Reply:
They are in a text block like they should be in order to avoid this problem. No escaping is needed.

Reply:
Yeah, we should really add a test that `cargo doc` completes without warnings

Reply:
Got it. Thanks.